### DISCUSSION TITLE
contentEditable not working

### DISCUSSION THREAD

**Discussion by saasira** [c-smile/sciter-js-sdk#191] [Q&A] [answered]
Hi,

It seems contentEditable feature  is not implemented yet by Sciter Engine; may we know if it will be implement at a later time, or or not in the scope at all?

Thanks,
Samba

**Comment by c-smile** [comment, marked as answer]
Sciter does not support contenteditable as it is quite limited feature. 

Instead, Sciter has special native behavior for WYSIWYG editing.

Please check sample: https://github.com/c-smile/sciter-js-sdk/tree/main/samples.sciter/editor-richtext
and documentation : https://github.com/c-smile/sciter-js-sdk/blob/main/docs/md/behaviors/behavior-richtext.md

These applications https://notes.sciter.com/ and https://html-notepad.com/ are both using that behavior.
